src/libertem/web/jobs.py

```diff
@@ -1,12 +1,15 @@
 import logging
```

Comment:
I think this is an artifact from rebasing - a lot of this code was moved to `libertem.web.engine` for #1172, so the changes in this PR need to be ported to that code. I think this was mostly getting the `UDFRunner` subclass from the executor?